core/store/orm/orm.go

```diff
@@ -816,6 +816,9 @@ func (orm *ORM) JobsSorted(sort SortType, offset int, limit int) ([]models.JobSp
 	var jobs []models.JobSpec
 	order := fmt.Sprintf("created_at %s", sort.String())
 	err = orm.getRecords(&jobs, order, offset, limit)
+	if len(jobs) == 0 && count != 0 {
```

Comment:
So I think if you navigate to a page, far beyond the record set, it's okay to return a 404. e.g: You have 10 records, you show 30 records per page, and you navigate to page 2: 404. But not to show a 404 for an empty page, i.e. you have 0 records and are on page 0.

Reply:
Yeah that captures exactly what I was thinking